src/parse/utilities.ts

```diff
@@ -15,7 +15,7 @@ export function getIndentation(line: string): string {
  */
 export function preprocessLines(lines: string[]): string[] {
     return lines
-        .map(line => line.trim())
+        .map(line => line.split("#")[0].trim())
         .filter((line) => !line.startsWith("#"));
```

Comment:
```suggestion
        .filter((line) => !line.startsWith("#"))
        .map(line => line.split("#")[0].trim());
```

Otherwise the condition `startsWith("#")` will never be true.

Reply:
Oof, I totally missed that, good catch!